variables.tf

```diff
@@ -25,7 +25,7 @@ variable "container_image_url" {
 variable "deployment_maximum_percent" {
   type        = number
   description = "The upper limit of the number of tasks (as a percentage of `desired_count`) that can be running in a service during a deployment"
-  default     = 100
+  default     = 150
```

Comment:
this will all of a sudden change the deployments of a lot of services we are running. Why is it needed? If it is due to the desired count... should we just run more replicas?

Reply:
only traefik